src/refract/RenderJSONVisitor.cc

```diff
@@ -102,10 +105,38 @@ namespace refract
         if (!a || a->value.empty()) {
             return NULL;
         }
+        
+        std::cerr << "samples" << std::endl;
 
         return TypeQueryVisitor::as<T>(*(a->value.begin()));
     }
 
+    template<typename T>
+    const T* getNullable(const T& e) {
+        IElement::MemberElementCollection::const_iterator ta = e.attributes.find("typeAttributes");
+        
+        if (ta == e.attributes.end()) {
+            return NULL;
+        }
+        
```

Comment:
`ta` is a `MemberElement`. But the issue here is that apparently the code is not coming here. I think `snowcrash` still has a problem since it keeps giving an error saying that ``unable to find `nullable` type``